File: JSBase/트리/ex.js
class Node {
  constructor(value = null) {
    this.value = value;
    this.left = null;
    this.right = null;
  }
}

var fs = require("fs");
// const input = fs.readFileSync('dev/stdin').toString().split('\n');
var input = fs.readFileSync("./ex.txt").toString().split("\r\n");

const N = parseInt(input[0]);
console.log(N);

File: 파이썬베이스/수학/최대공약수_최소공배수/ex.py
import sys
sys.stdin = open("2609.txt", "r")
input = sys.stdin.readline


def gcd(x, y):
    if y == 0:
        return x
    else:
        return gcd(y, x % y)


N, M = map(int, input().split())
g = gcd(N, M)
print(g)
print(N * M//g)
